### ISSUE TITLE
Outdated dependency on analyzeme

### ISSUE THREAD

**Issue by bjorn3** [rust-lang/rustc-perf#857]
When trying to use `collector profile_local self-profile` I get the following panic:

```
thread 'main' panicked at 'unexpected file "results/Zsp-cg_llvm-deep-vector-Check-Full/issue_20936_deep_vector-5231.mm_profdata"', collector/src/execute.rs:884:25
stack backtrace:
   0: rust_begin_unwind
             at /rustc/52e3dffa50cfffdcfa145c0cc0ba48b49abc0c07/library/std/src/panicking.rs:493:5
   1: std::panicking::begin_panic_fmt
             at /rustc/52e3dffa50cfffdcfa145c0cc0ba48b49abc0c07/library/std/src/panicking.rs:435:5
   2: <collector::execute::ProfileProcessor as collector::execute::Processor>::process_output
             at ./collector/src/execute.rs:884:25
   3: collector::execute::CargoProcess::run_rustc
             at ./collector/src/execute.rs:424:23
   4: collector::execute::Benchmark::measure
             at ./collector/src/execute.rs:1242:21
   5: collector::main_result
             at ./collector/src/main.rs:763:21
   6: collector::main
             at ./collector/src/main.rs:441:11
   7: core::ops::function::FnOnce::call_once
             at /home/bjorn/.rustup/toolchains/nightly-2021-03-26-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/function.rs:227:5
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```

**Comment by bjorn3** [comment]
When I fix that function to only consider `.mm_profdata` as it should with the current measureme version I get:

```
Profiling with SelfProfile
8 benchmarks remaining
Preparing deep-vector
Running deep-vector: Check + [Full, IncrFull, IncrUnchanged, IncrPatched]
thread 'main' panicked at 'couldn't read string_data file: Os { code: 2, kind: NotFound, message: "No such file or directory" }', analyzeme/src/profiling_data.rs:49:60
stack backtrace:
   0: rust_begin_unwind
             at /rustc/7402a394471a6738a40fea7d4f1891666e5a80c5/library/std/src/panicking.rs:483
   1: core::panicking::panic_fmt
             at /rustc/7402a394471a6738a40fea7d4f1891666e5a80c5/library/core/src/panicking.rs:85
   2: core::option::expect_none_failed
             at /rustc/7402a394471a6738a40fea7d4f1891666e5a80c5/library/core/src/option.rs:1221
   3: analyzeme::profiling_data::ProfilingData::new
   4: flamegraph::main
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
mv: kan status van 'rustc.svg' niet opvragen: Bestand of map bestaat niet
collector error: Failed to profile 'deep-vector' with SelfProfile, recorded: mv "rustc.svg" "results/flamegraph-cg_llvm-deep-vector-Check-Full": ExitStatus(ExitStatus(256))

Stack backtrace:
   0: anyhow::private::new_adhoc
             at /home/bjorn/.cargo/registry/src/github.com-1ecc6299db9ec823/anyhow-1.0.31/src/lib.rs:610:36
      collector::execute::rename
             at ./collector/src/execute.rs:40:13
   1: <collector::execute::ProfileProcessor as collector::execute::Processor>::process_output
             at ./collector/src/execute.rs:897:17
   2: collector::execute::CargoProcess::run_rustc
             at ./collector/src/execute.rs:424:23
   3: collector::execute::Benchmark::measure
             at ./collector/src/execute.rs:1238:21
   4: collector::main_result
             at ./collector/src/main.rs:763:21
   5: collector::main
             at ./collector/src/main.rs:441:11
   6: core::ops::function::FnOnce::call_once
             at /home/bjorn/.rustup/toolchains/nightly-2021-03-26-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/function.rs:227:5
      std::sys_common::backtrace::__rust_begin_short_backtrace
             at /home/bjorn/.rustup/toolchains/nightly-2021-03-26-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/std/src/sys_common/backtrace.rs:125:18
   7: std::rt::lang_start::{{closure}}
             at /home/bjorn/.rustup/toolchains/nightly-2021-03-26-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/std/src/rt.rs:66:18
   8: core::ops::function::impls::<impl core::ops::function::FnOnce<A> for &F>::call_once
             at /rustc/52e3dffa50cfffdcfa145c0cc0ba48b49abc0c07/library/core/src/ops/function.rs:259:13
      std::panicking::try::do_call
             at /rustc/52e3dffa50cfffdcfa145c0cc0ba48b49abc0c07/library/std/src/panicking.rs:379:40
      std::panicking::try
             at /rustc/52e3dffa50cfffdcfa145c0cc0ba48b49abc0c07/library/std/src/panicking.rs:343:19
      std::panic::catch_unwind
             at /rustc/52e3dffa50cfffdcfa145c0cc0ba48b49abc0c07/library/std/src/panic.rs:431:14
      std::rt::lang_start_internal
             at /rustc/52e3dffa50cfffdcfa145c0cc0ba48b49abc0c07/library/std/src/rt.rs:51:25
   9: main
  10: __libc_start_main
  11: _start
7 benchmarks remaining
```

**Comment by tgnottingham** [comment]
Same as #850 -- looks like it was fixed by commit 4fa8ee6362bdfeacc03781f484ee4f9593326bbe of #865.

**Comment by bjorn3** [comment]
Looks like the flamegraph links on the query details page still give a 500 internal server error.

**Comment by tgnottingham** [comment]
The flamegraph links are now fixed by #978. I think this issue is solved, so I'll close. Please reopen if I missed something.
